extra/import_data.py

```diff
@@ -111,9 +111,34 @@ def upload_to_mongodb(inheritance_analysis, accession_number, analyis_sources, p
         else:
             logger.info("error, externalsources has changed format in Alissa")
     patient["annotation_sources"] = externalSources_dict  # add previous section to total patient info
-    
-    # add VUS info to patientdata
-    for variant in variants:
+
+    # extract information from the VUS/variant data and add to "patient"
+    for variant in vus_export:
+        # format for gnomad Links. 4 availble in Alissa (snp, insertion, deletion and substitution)
+        fullgnomen = variant['platformDatasets']['HGVS genomic-level nomenclature (fullGNomen)']  # NC_000001.10:g.123456789T>A
+        variant['fullgnomen'] = fullgnomen
+        if fullgnomen:  # NC_000001.10:g.123456789T>A
+            gnomad_data = re.split(':[a-z].', fullgnomen)[1]  # 123456789T>A
+            if variant["type"] == "snp":
+                gnomad_data = [c for c in re.split(r'([-+]?\d*\.\d+|\d+)', gnomad_data) if c]  # 123456789T>A
+                gnomad_data = variant["chromosome"] + "-" + re.sub('[<>]+', '-', ("-".join(gnomad_data)))  # 1-123456789-T-A
+                variant['GnomadVariant'] = {'Single nucleotide variant': gnomad_data}
+            elif variant["type"] == "insertion":
+                gnomad_data = variant["chromosome"] + "-" + gnomad_data
+                variant['GnomadVariant'] = {'Insertion': gnomad_data}
+                # TODO: make link format correctly for this genomic variation
+            elif variant["type"] == "deletion":
+                gnomad_data = variant["chromosome"] + "-" + gnomad_data
+                variant['GnomadVariant'] = {'Deletion': gnomad_data}
+                # TODO: make link format correctly for this genomic variation
+            elif variant["type"] == "substitution":
+                gnomad_data = variant["chromosome"] + "-" + gnomad_data
+                variant['GnomadVariant'] = {'Substitution': gnomad_data}
+                # TODO: make link format correctly for this genomic variation
```

Comment:
yes, because I made it generic now, but want to split the naming of the `variant['GnomadVariant']` to either of the 4 possible SNVs, to later distinguish them without using too much HTML formatting. However, due to open ticket at Agilent (Alissa) I cannot program them now. To be short, there is no left-aligned data in Alissa at the moment. For a version 1.0 it was decided that this is not yet needed. Most VUS are snp anyway and users can manually search the GnomAD database if necessary.